common/static/common/js/spec/main_requirejs.js

```diff
@@ -1,6 +1,11 @@
-(function(requirejs, define) {
+/* globals _, requirejs */
+/* eslint-disable quote-props */
```

Comment:
Ok, that's a legit reason - requirejs files have lots of library names as keys that are kebab cased, whereas normally our objects would have camelCased keys :thumbsup: